Bigtable/src/Instance.php

```diff
@@ -0,0 +1,300 @@
+<?php
+/**
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\Admin\V2\BigtableInstanceAdminClient as InstanceAdminClient;
+use Google\Cloud\Bigtable\Admin\V2\Instance_Type;
+use Google\Cloud\Bigtable\Admin\V2\Instance_State;
+use Google\Cloud\Bigtable\Admin\V2\StorageType;
+use Google\Cloud\Bigtable\Connection\ConnectionInterface;
+use Google\Cloud\Core\ArrayTrait;
+use Google\Cloud\Core\Exception\NotFoundException;
+use Google\Cloud\Core\Iterator\ItemIterator;
+use Google\Cloud\Core\LongRunning\LongRunningConnectionInterface;
+use Google\Cloud\Core\LongRunning\LongRunningOperation;
+use Google\Cloud\Core\LongRunning\LROTrait;
+
+/**
+ * Represents a Cloud Bigtable instance
+ *
+ * Example:
+ * ```
+ * use Google\Cloud\Bigtable\BigtableClient;
+ *
+ * $bigtable = new BigtableClient();
+ *
+ * $instance = $bigtable->instance('my-instance');
+ * ```
+ *
+ * @method resumeOperation() {
```

Comment:
Check if this is really needed (seems @sduskis  says this is not needed in CBT).
+ If it's needed, I don't think this syntax for showing `@method` is quite correct.